External/FEXCore/Source/Utils/Allocator.cpp

```diff
@@ -240,6 +248,22 @@ namespace FEXCore::Allocator {
           STEAL_LOG("[%d] ParseEnd; RegionBegin: %016lX RegionEnd: %016lX\n", __LINE__, RegionBegin, RegionEnd);
 
           State = ScanEnd;
+
+          // If the previous map's ending and the region we just parsed overlaps the stack then we need to save the stackmapping.
+          // Otherwise we will only have ~144KB stack size which crashes quickly.
```

Comment:
Where does the 144kB value actually come from?

Reply:
From me inspecting a crash from thunks where that was the amount of stack space used up to that point of crash.
Can change the comment to some non-constant value like "We will only have an itty-bitty stack size..."

Reply:
Changed comment to not explicitly talk about a stack size, which will be variable.